go.mod

```diff
@@ -1,11 +1,13 @@
 module github.com/golangci/golangci-lint
 
+go 1.11
```

Comment:
Why does this PR specify `go 1.11`? I actively don't want this.

Reply:
Could you explain what issue this raises for you?

This statement is not a hard requirement. It's merely an indication used by the `go` toolchain and you should not need to worry about it.

Tl;dr: This tells the Go compiler that if the build fails and you are using a < 1.11 version it will tell you it might be due to a too-low Go version... however given that versions < 1.11 are not module aware they will not even bother reading this.

Reply:
For reference see the [Go 1.12 release notes](https://golang.org/doc/go1.12#modules).

Reply:
My understanding is that this specifier causes the files in the module to be built with an older version of Go?

As mentioned here: https://golang.org/doc/go1.12#modules

> The go directive in a go.mod file now indicates the version of the language used by the files within that module.

Reply:
It's actually a bit different.

> the version of the language used by the files within that module

refers to the version for which the code was written (as it might be using a new function introduced in that version... but not necessarily), not the one with which the files should be built. As the sentence following the one you quote indicates:

>  If the go directive for a module specifies a version newer than the toolchain in use, the go command will attempt to build the packages regardless, and will note the mismatch only if that build fails.

Reply:
I still feel that the writing is a bit ambiguous here, but I think I see what you're saying. I don't object if it's true that it's only a guideline. I just don't want tools like this to force-build on an old version, if that won't happen, I retract my objection :-)

Reply:
You will always have full control over your build process. There is no version enforcement going, I can guarantee you that. 🙂 

I'll forward the point about the feeling of ambiguity around the phrasing. I can imagine myself in your place and understand it can introduce confusion for people not familiar with the relatively new syntax of `go.mod` files.